src/tools/default_flags_table/reco_flags.py

```diff
@@ -74,7 +74,100 @@
     ('BEMC:EcalBarrelTruthClusters:moduleDimZName',              '',                               ''),
     ('BEMC:EcalBarrelTruthClusters:samplingFraction',            '1',                              '*'),
 
-    # EEMC - Endcap EMC
+    # BEMC - Imaging Barrel
+    #------------------
+    ('BEMC:EcalBarrelImagingRawHits:input_tag',                 'EcalBarrelHits',                  'Name of input collection to use'),
+    ('BEMC:EcalBarrelImagingRawHits:energyResolutions',         '0.0*MeV',                         ''),
```

Comment:
Should be ``[0.0, 0.02, 0.0]`` - 2% flat resolution https://eicweb.phy.anl.gov/EIC/benchmarks/physics_benchmarks/-/blob/master/options/reconstruction.py#L571

Reply:
Fixed.